core/server/common/src/main/java/alluxio/master/journal/checkpoint/CheckpointType.java

```diff
@@ -43,7 +43,11 @@ public enum CheckpointType {
   /**
    * A checkpoint consisting of a single long value written by a data output stream.
    */
-  LONG(5, new LongCheckpointFormat());
+  LONG(5, new LongCheckpointFormat()),
+  /**
+   * A RocksDB backup in .zip format.
+   */
+  ROCKS_ZIP(6, new ZipCheckpointFormat());
```

Comment:
@tcrain what do you say we change this to `ROCKS_SINGLE` and `ROCKS_PARALLEL`?

Reply:
Sure that sounds good.

Reply:
@adol001 could you please change the RocksDB checkpoint types to `ROCKS_SINGLE` and `ROCKS_PARALLEL`? Thx!